dev/com.ibm.ws.jaxws.2.3.clientcontainer/src/com/ibm/ws/jaxws/ConduitConfigurer.java

```diff
@@ -0,0 +1,213 @@
+/**
```

Comment:
This code is identical to what's in the com.ibm.ws.jaxws.2.3.common bundle, but I intend to refactor all of the jaxws.2.x.common/jaxws.2.x.clientcontainer bundles to remove duplicate code.